packages/website/src/components/ClientOnly/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from 'react'
+
+type Props = {
+  children: React.ReactElement
+}
+
+const ClientOnly: React.FC<Props> = (props) => {
```

Comment:
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

```suggestion
const ClientOnly = (props) => {
```